crowbar_framework/spec/models/api/crowbar_spec.rb

```diff
@@ -125,6 +126,36 @@
     end
   end
 
+  context "with enough compute resources" do
+    it "succeeds to find enough compute nodes" do
+      allow(NodeObject).to(
+        receive(:find).with("roles:nova-compute-kvm").
+        and_return([:node, :node])
```

Comment:
it passes, because it doesn't perform anything on the NodeObject, just the size of the Array. so this is wrong